docs/accessing-the-cluster.md

```diff
@@ -203,5 +301,40 @@ monitoring-influx-grafana-controller-0133o   10.244.3.4          influxdb
                                                                  grafana                 kubernetes/heapster_grafana:v0.4
 ```
 
+##<a name="somanyproxies"></a>So Many Proxies
+There are several different proxies you may encounter when using kubernetes:
+  1. The [kubectl proxy](#kubectlproxy)
+    - runs on a users desktop or in a pod
+    - proxies from a localhost address to the kubernetes apiserver.
+    - adds authentication headers
+    - does not provide load balancing in the future.
+    - client to proxy uses HTTP
+    - proxy to apiserver uses HTTPS
+    - locates apiserver
+  1. The [proxy verb](#proxyverb) is a collection of urls which cause the kubernetes apiserver to act as a proxy.
+    - runs in the apiserver processes
+    - client to proxy uses HTTPS (or http if apiserver so configured)
+    - proxy to target may use HTTP or HTTPS as chosen by proxy using available information
+    - target may be a port on a Node, Pod, or Service.
+    - Does not itself provide load balancing.
```

Comment:
This is not true-- if you use a service, it does loadbalancing across all the available endpoints.